src/Altinn.Notifications/Models/SmsNotificationOrderRequestExt.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Altinn.Notifications.Models;
+
+/// <summary>
+/// Class representing an SMS notiication order request
+/// </summary>
+/// <remarks>
+/// External representaion to be used in the API.
```

Comment:
```suggestion
/// External representation to be used in the API.
```
Please fix the typo in the email class as well